Project.toml

```diff
@@ -33,7 +34,7 @@ Xorg_xkeyboard_config_jll = "2.27.0"
 adwaita_icon_theme_jll = "3.33.92"
 gdk_pixbuf_jll = "2.38.2"
 hicolor_icon_theme_jll = "0.17.0"
-julia = "1.3"
+julia = "1.6"
```

Comment:
Reminder for the maintainers of the package that narrowing the range of supported versions of Julia warrants a minor version bump